### PR TITLE
lambda function call moved out

### PR DESCRIPTION
**Pull request by jyleong**
james + kai
* lambda function call moved out to another import
* make decision on what app to call

### CODE REVIEW

File: handler.py [hunk 1]
```diff
@@ -4,7 +4,14 @@
 
 RASASERVER_URL = "http://ec2-34-218-219-244.us-west-2.compute.amazonaws.com:5000/parse"
 
+APP_DICT = {
+    "cats": invokeLamba,
+    "home": None,
```
Comment (kevinzhangftw) [inline handler.py]: this can be removed because home is not app, this can be just a function call

File: handler.py [hunk 2]
```diff
@@ -30,28 +38,15 @@ def wrapIntentSpeakAction(rasaJson, sender):
...
     payload, sender = unwrapEvent(event)
     rasaJson = postRasaForIntent(payload)
-
-    return callCatApp()
-    #return wrapIntentSpeakAction(rasaJson, sender)
+    intentName = rasaJson['intent']['name']
+    if intentName == 'cats':
+        return APP_DICT[intentName]('catApp', payload)
+    elif intentName == 'home':
+        return wrapIntentSpeakAction(rasaJson, sender)
```
Comment (kevinzhangftw) [inline handler.py:52]: Stubbed function: here we want to tell user we are home now. and then lambda function should recursively invoke it self

**Review by kevinzhangftw — COMMENTED** [review]
need to vOS flow

File: handler.py (hunk 1 above)
Comment (kevinzhangftw) [reply handler.py]: on the second thought this should be a recursive call.

Comment (kimjune01) [reply handler.py]: APP_DICT isn't descriptive enough of a name

**Pull request merged**